test/messages-test.js

```diff
@@ -1,30 +1,40 @@
 import async from 'async'
-var messages = require('../scripts/messages.js').messages
+import chai from 'chai'
+import sinon from 'sinon'
+let messages = require('../scripts/messages.js').messages
 
 
-describe('auth and messages test', function () {
-    it('authenticate', function () {
-        async.series([
-            function setAuth(step) {
-                step(null, messages.setAuth())
+describe('Google Docs and Gists Logging', function () {
+    it('Google Docs messaging', function () {
+        let message = {
+            "text": "+todo testing @test",
+            "user": {
+                "name": "test"
             }
-        ])
-    })
-    it('message send', function () {
-        async.series([
-            function setAuth(step) {
-                messages.setAuth()
-            },
-            function sendMessage(step) {
-                    step(null, messages.sendMessage({
-                        text: "+todo @assignees some text",
-                        user: {
-                            name: "Artem (weirdGuy)",
-                            login: "weirdGuy"
-                        }
-                    }))
-            },
-        ])
+        }
+        sinon.stub(messages, "sendMessage", function () {
+            return "0"
+        })
+
+        messages.sendMessage(message, "dest", function (inf) {})
+
+        sinon.assert.calledOnce(messages.sendMessage)
+
     })
 
+    it('Gist messaging', function () {
+        let message = {
+            "text": "+todo testing @test",
+            "user": {
+                "name": "test"
+            }
+        }
+        sinon.stub(messages, "sendGist", function () {
```

Comment:
You need to stub at a deeper level than this. As it currently stands this test is pointless as it does not test anything.

You would really want to stub the method that does the call out to the github gist API if you could.